src/Parser/Core/Modules/NetherlightCrucibleTraits/NLCTraits.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import StatisticsListBox, { STATISTIC_ORDER } from 'Main/StatisticsListBox';
+
+import Module from 'Parser/Core/Module';
+
+// import CoalescingMists from './CoalescingMists';
+import MurderousIntent from './MurderousIntent';
+import RefractiveShell from './RefractiveShell';
+import Shocklight from './Shocklight';
+import SecureInTheLight from './SecureInTheLight';
+import InfusionOfLight from './InfusionOfLight';
+import LightsEmbrace from './LightsEmbrace';
+import Shadowbind from './Shadowbind';
+import ChaoticDarkness from './ChaoticDarkness';
+import TormentTheWeak from './TormentTheWeak';
+import DarkSorrows from './DarkSorrows';
+
+class RelicTraits extends Module {
+  static dependencies = {
+    // coalescingMists: CoalescingMists,
+    murderousIntent: MurderousIntent,
+    refractiveShell: RefractiveShell,
+    shocklight: Shocklight,
+    secureInTheLight: SecureInTheLight,
+    infusionOfLight: InfusionOfLight,
+    lightsEmbrace: LightsEmbrace,
+    shadowbind: Shadowbind,
+    chaoticDarkness: ChaoticDarkness,
+    tormentTheWeak: TormentTheWeak,
+    darkSorrows: DarkSorrows,
+  };
+
+  statistic() {
+    return (
+      <StatisticsListBox
+        title="Netherlight Crucible"
+        tooltip="This provides an overview of the increased provide by the Netherlight Crucible traits."
+        style={{ minHeight: 186 }}
```

Comment:
It might be best to not set a min height, especially not this high, since it often won't reach near that height. Besides with Masonry aligning the tiles properly now we don't have to worry about height anymore.